check-reserve-update-frequency/bot-config.json

```
@@ -0,0 +1,10 @@
+{
+  "developerAbbreviation": "AE",
+  "protocolName": "Aave",
+  "protocolAbbreviation": "AAVE",
+  "oracleAgeThresholdSeconds": 86400,
+  "contractAddresses": {
+    "LendingPoolAddressesProvider": "0xB53C1a33016B2DC2fF3653530bfF1848a515c8c5",
+    "ProtocolDataProvider": "0x057835Ad21a177dbdd3090bB1CAE03EaCF78Fc6d"
+  }
+}
```

Any interest in formatting this similarly to the other `bot-config.json` files?

I realize this would require reworking the imports in the agent despite not changing its functionality.
```suggestion
{
  "developerAbbreviation": "AE",
  "protocolName": "Aave",
  "protocolAbbreviation": "AAVE",
  "oracleAgeThresholdSeconds": 86400,
  "contracts": {
    "LendingPoolAddressesProvider": {
      "address": "0xB53C1a33016B2DC2fF3653530bfF1848a515c8c5",
      "abiFile": "ILendingPoolAddressesProvider.json",
    }
    "ProtocolDataProvider": {
      "address": "0x057835Ad21a177dbdd3090bB1CAE03EaCF78Fc6d",
      "abiFile": "AaveProtocolDataProvider.json"
    }
  }
}
```

---

Actually, looking at the Bot code, it is quite far from our current conventions (e.g. all caps snakecase for constants, loading ABIs in global scope instead of initialize(), etc.).  I'll leave it up to you as to whether the time should be spent reworking the code for whatever little benefit in readability and consistency would be gained.